wildfly-microprofile-rest-archetype/src/main/resources/archetype-resources/src/main/java/SampleEndPoint.java

```diff
@@ -0,0 +1,16 @@
+package ${package};
+import java.util.Date;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/Sample")
+public class SampleEndPoint {
+
+    @GET
+	@Path("/helloworld")
+	public String helloworld() {
+        return "Hello World!\n " + "now is: " + new Date();
+	}
```

Comment:
I actually like this minimal example more than the simple app (creation date, current date) in the Jakarta EE archetype. But that might be only my preference.